### PR TITLE
add some detail to reporting for user/group mods (NEED HELP)

### PR DESCRIPTION
**Pull request by bby-bishopclark**
Signed-off-by: Bishop Clark <bishopolis@gmail.com>

### Description

This minor patch provides added detail around user and group changes.  Instead of 
 - would alter group replace group members with new list of members

This change reports 
 - would alter group replace group members ([dave, bob]) with new list of members ([bob, doug])

Similar detail was added to converge_by messages for user mods (uid, homedir, etc)

Just messages changes.

### Check List

- [x] New functionality includes tests
- [ ] All tests pass
- [x] RELEASE\_NOTES.md has been updated if required (not required for bugfixes, required for API changes)
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

**Comment by bby-bishopclark** [comment]
Yeah, I see the bug, some dumb mistake, copypasta when I wasn't looking.  Stand by.  I'll add in the missing tests that I can as well.

**Comment by tas50** [comment]
This would be a good one for msys to complete

File: lib/chef/provider/user.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ class Chef
   class Provider
     class User < Chef::Provider
 
-      attr_accessor :user_exists, :locked
+      attr_accessor :user_exists, :locked, :change_desc
```
Comment (tas50) [inline lib/chef/provider/user.rb]: This probably isn't necessary

**Comment by tas50** [comment]
I'm going to close this PR out at this point since we haven't heard back in some time.

**Comment by bby-bishopclark** [comment]
Thanks Tim.  I've not paid attention to this project once the terminal release for v14 was announced.  I'll rejigger it for cinc if it'll fit there, as that's where we're going.

**Pull request closed without merging**